dvc/repo/artifacts.py

```diff
@@ -0,0 +1,44 @@
+import os
+from typing import TYPE_CHECKING, Any, Dict
+
+from dvc.annotations import Artifact
+from dvc.dvcfile import FileMixin
+from dvc.utils import relpath
+
+if TYPE_CHECKING:
+    from dvc.repo import Repo
+
+
+class ArtifactsFile(FileMixin):
+    from dvc.schema import ARTIFACTS_SCHEMA as SCHEMA
```

Comment:
Did you mean `SINGLE_ARTIFACT_SCHEMA` or is `str` also supported inside the file?

Reply:
Yes, `str` also supported - it gets unraveled from a separate file, see [this test](https://github.com/aguschin/dvc/blob/artifacts/tests/func/artifacts/test_artifacts.py#L72).

Reply:
Can `artifacts.yaml` only have a string like this? I understand that it may be a string in `dvc.yaml:artifacts`, but this is ArtifactsFile right? 
```yaml
# artifacts.yaml
string
```

Reply:
Fixed. Thanks for the review - it was very useful 🙏🏻